Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Name": "Digital Shadows",
+    "Author": "Digital Shadows - support@digitalshadows.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/DigitalShadowsLogo.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster.",
+    "Data Connectors": [
+      "Data Connectors/Digital Shadows/DigitalShadowsSearchlight_API_functionApp.json"
+    ],
+    "Analytic Rules": [
+      "Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml",
+      "Analytic Rules/Digital_Shadows_incident_creation_include.yaml"
+    ],
+    "Workbooks": [
+        "Workbooks/DigitalShadows.json"
+    ],
+    "Playbooks": [
+        "Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json"
+    ],
+    "Metadata": "SolutionMetadata.json",
+    "BasePath": "/home/metron/Desktop/Github/Azure-Sentinel/Solutions/Digital Shadows",
```

Comment:
Make the base path changes as required.
@kapil-metron Please take a look at this.